### PR TITLE
Paginate API incidents

### PR DESCRIPTION
**Pull request by joecohens**
//cc @jbrooksuk 

Closes https://github.com/cachethq/Cachet/issues/253

### CODE REVIEW

File: src/Repositories/EloquentRepository.php [hunk 1]
```diff
@@ -18,6 +18,16 @@ public function all()
     }
 
     /**
+     * Returns paginated result.
+     *
+     * @return \Illuminate\Pagination\Paginator
+     */
+    public function paginate($per_page = 20)
```
Comment (GrahamCampbell) [inline src/Repositories/EloquentRepository.php]: perPage?

Comment (joecohens) [inline src/Repositories/EloquentRepository.php]: Lol yeah thanks!

**Comment by jbrooksuk** [comment]
@joecohens rebase and merge :+1:

**Comment by GrahamCampbell** [comment]
@jbrooksuk Do you want to switch over to the new styleci btw?

**Comment by GrahamCampbell** [comment]
Delete the webhook from here, and then login to https://styleci.io/, and enable it on your account page.

**Comment by GrahamCampbell** [comment]
Actually, I have admin access, I can do it if you like?

**Comment by jbrooksuk** [comment]
Done @GrahamCampbell.

**Pull request merged**